client/src/components/Routes/PrivateRoute.js

```diff
@@ -1,10 +1,15 @@
-import React, { useEffect, useState } from "react";
-import { Route, Redirect } from "react-router-dom";
+import React from "react";
+import PropTypes from "prop-types";
+import { Route } from "react-router-dom";
 import { useAuth } from "../../providers/AuthProvider";
 
 const PrivateRoute = ({ children, ...rest }) => {
   const authenticated = useAuth();
   return authenticated && <Route {...rest}>{children}</Route>;
 };
 
+PrivateRoute.PropTypes = {
```

Comment:
`PrivateRoute.PropTypes` is specific to class components in React, for functional components you have to use `PrivateRoute.propTypes`

You still use `PropTypes.any` for example though

Reply:
Huh, I tried that and eslint didn't recognize `propTypes`, but it seems to work now!